.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.11.3
+    steps:
+      - checkout 
+      - run:
+          name: Update npm
+          command: 'sudo npm install -g npm@latest'
```

Comment:
node v8 should have a recent enough version of npm to avoid having to update it.